VideoLocker/src/main/java/org/edx/mobile/model/api/AccessError.java

```diff
@@ -0,0 +1,19 @@
+package org.edx.mobile.model.api;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.HashMap;
+
+/**
+ * Created by lindaliu on 7/13/15.
+ */
+
+public enum AccessError {
+    @SerializedName("course_not_started")
+    StartDateError,
+    @SerializedName("not_visible_to_user")
+    VisibilityError,
+    @SerializedName("unfulfilled_milestones")
+    MilestoneError,
+    UnknownError;
```

Comment:
Were you able to figure out what gson does if it doesn't recognize any of the enum names? I went googling around a bit and couldn't find anything. They don't even have a test case for it https://github.com/google/gson/blob/master/gson/src/test/java/com/google/gson/functional/EnumTest.java